src/txmempool.h

```diff
@@ -599,6 +601,14 @@ class CTxMemPool
      */
     explicit CTxMemPool(CBlockPolicyEstimator* estimator = nullptr, int check_ratio = 0);
 
+    // Set CDeterministicMNManager pointer as it's initialized after
```

Comment:
should be doxygen format of comments here too

Reply:
Resolved in latest push.